backend/Squirrel.ConsoleApp.BL/Services/ConnectionFileService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models.Models;
+
+namespace Squirrel.ConsoleApp.BL.Services;
+
+public class ConnectionFileService: IConnectionFileService
+{
+    public void CreateEmptyFile()
+    {
+        var filePath = GetFilePath();
+        if (!File.Exists(filePath))
+        {
+            File.WriteAllText(filePath, "{}");
+        }
+    }
+
+    public ConnectionString ReadFromFile()
+    {
+        var filePath = GetFilePath();
+        if (File.Exists(filePath))
+        {
+            var json = File.ReadAllText(filePath);
+            return JsonConvert.DeserializeObject<ConnectionString>(json) ?? throw new InvalidOperationException();
```

Comment:
Add a message to exception ('failed to read json from file' or something like this), it's not clear what has gone wrong. Or you can even create your custom exception